packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -9,3 +9,4 @@ export * from './lib/components';
 export * from './lib/primitives';
 export * from './lib/services';
 export * from './lib/directives/amplify-override.directive';
+export * from './lib/directives/proxies';
```

Comment:
Very very minor, but can we move the proxy file to somewhere else? `lib/directives` is namespaced for custom Angular directives. Maybe "./lib/proxies" can work, but I'll leave that up to you.

Reply:
ok I'll move the file to the root of the `lib` folder